Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -879,6 +890,23 @@ async def main():
 
         self.loop.run_until_complete(main())
 
+    def test_subprocess_break_pipe(self):
+        # See https://github.com/python/cpython/issues/130925
+        async def main():
+            proc = await asyncio.create_subprocess_exec(*PROGRAM_YES,
+                                                        stdout=asyncio.subprocess.PIPE)
+            try:
+                # just make sure the program has executed correctly
```

Comment:
```suggestion
```

Reply:
I added that comment so it's clear why those otherwise unnecessary lines of code are there. They aren't conceptually part of the unit test though. I can be more clear.